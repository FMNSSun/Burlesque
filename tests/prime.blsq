,99ro?ifC)n!p^1