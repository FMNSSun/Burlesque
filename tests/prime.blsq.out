1
2
3
2
5
3
7
2
3
5
11
2
13
7
5
2
17
3
19
2
7
11
23
2
5
13
3
2
29
5
31
2
11
17
7
3
37
19
13
2
41
7
43
2
3
23
47
2
7
5
17
2
53
3
11
2
19
29
59
2
61
31
3
2
13
11
67
2
23
7
71
2
73
37
5
2
11
13
79
2
3
41
83
2
17
43
29
2
89
3
13
2
31
47
19
2
97
7
3
5